src/index.ts

```diff
@@ -40,7 +40,7 @@ async function run() {
 
     const output = await getExecOutput(
         'dvc',
-        ['diff', `origin/${baseBranch}...origin/${headBranch}`, '--format', 'markdown', ...prLinkArgs, ...authArgs]
+        ['diff', `origin/${baseBranch}...origin/${headBranch}`, '--format', 'markdown', ...prLinkArgs, ...authArgs, '--caller', 'github.pr_insights']
```

Comment:
we'll have to update the cli version to 4.2.10 too so it can accept the correct caller flag

Reply:
updated